package.json

```diff
@@ -112,6 +112,7 @@
     "docs": "jsdoc -R README.md -d docs/ -r src/",
     "lint": "eslint .",
     "test": "eslint . && stylelint src/**/*.scss && jest",
+    "jest": "jest",
```

Comment:
this is just a passthrough to npx right?

Reply:
I believe it uses the jest that's installed locally. It's using the same one as the test command. I just found it quicker for testing to not have to run the linter every time

Reply:
I think running `npx jest` will do the exact same thing though (I also like to skip the linter!)

Reply:
oh, I see what you mean. we don't really need this because I can just use npx instead. I'll remove!

Reply:
removed!